09_abandoned_isles.conf

```
@@ -0,0 +1,12 @@
+team.1.color = red
+team.1 = red
+team.1.pos = (-80,-1,91)
+team.2.pos = (85,-1,-101)
+team.2.color = blue
+author = gimp and caoutiouspotato
+rotation = x
+h = 140
+team.2 = blue
+name = Abandoned Isles
+initial_stuff = default_sword:steel,shooter:pistol,default:sandstone 50
```

Are you sure this is correct?

```suggestion
initial_stuff = default:sword_steel,shooter:pistol,default:sandstone 50
```

---

Whoops, fixed it.